mandatory/4-tax.js

```diff
@@ -5,8 +5,13 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
 
+const TAX_RATE = 0.2;
+
+function calculateSalesTax(price) {
+  let tax = price * TAX_RATE;
```

Comment:
HI, for the  variable TAX_RATE  we can use camelcase like - taxRate. or you can avoid declaring the variable as well for that the code should be - function calculateSalesTax(price) {
  let salestax = (price * 0.2)
  return price + salestax;
} 

well done. Keep it up